+nigeLab/@Animal/subsref.m

```diff
@@ -39,186 +39,294 @@
+            % % %          if numel(subs) == 1
+            % % %             subs = subs{:};
+            % % %
+            % % %             % If only referencing child objects using a vector
+            % % %             % (not referencing animalObj, since animalObj is
+            % % %             %  already a scalar!)
+            % % %             if size(subs,2) == 1
+            % % %                S = substruct('{}',{1, subs});
+            % % %                varargout = {subsref(animalObj,S)};
+            % % %                return;
+            % % %
+            % % %                % Otherwise, if using a matrix to reference
+            % % %             elseif size(subs,2) == 2
+            % % %                S = substruct('{}',{subs(:,1),subs(:,2)});
+            % % %                varargout = {subsref(animalObj,S)};
+            % % %                return;
+            % % %
+            % % %                % Otherwise, could be using 'end'
+            % % %             else
+            % % %                if ~ischar(subs)
+            % % %                   error(['nigeLab:' mfilename ':badReference'],...
+            % % %                      'Matrix references should be nChild x 2');
+            % % %                end
+            % % %                if strcmpi(subs,'end')
+            % % %                   varargout = {animalObj.Block(...
+            % % %                      animalObj.getNumBlocks)};
+            % % %                   return;
+            % % %                else
+            % % %                   error(['nigeLab:' mfilename ':badReference'],...
+            % % %                      'Unrecognized index: %s',subs);
+            % % %                end
+            % % %             end
+            % % %
+            % % %             % Otherwise, subscript for Animal and Block both given
+            % % %          elseif numel(subs) == 2
+            % % %             if ~ischar(subs{1})
+            % % %                if any(subs{1} > 1) % since this is a scalar animalObj
+            % % %                   error(['nigeLab:' mfilename ':indexOutOfBounds'],...
+            % % %                      'Bad indexing expression, animalObj is scalar.');
+            % % %                end
+            % % %             end
+            % % %             S = substruct('()',{ones(size(subs,1),1),subs{2}});
+            % % %             varargout = {subsref(animalObj.Children,S)};
+            % % %             return;
+            % % %
+            % % %             % Otherwise, too many subscript args were given
+            % % %          else
+            % % %             error(['nigeLab:' mfilename ':tooManyInputs'],...
+            % % %                'Too many subscript indexing args (%g) given.',...
+            % % %                numel(subs));
+            % % %          end
+            % % %
+            % If more than one animalObj in array
+        else
```

Comment:
I would just double-check something you put on the `docs`. I've ported a lot of your comments to the `wiki` (**[here](https://github.com/m053m716/ePhys_packages/wiki/Startup_Shortcuts#vector-case)**). Specifically, take a look at that section; at least when I reference it using the `()` indexing it does still work, I am just confused by the `docs` I guess.

Reply:
I've updated the `wiki` to what I think it should be.